src/plugins/discover/public/application/apps/main/utils/update_search_source.ts

```diff
@@ -54,10 +54,7 @@ export function updateSearchSource(
 
     // this is not the default index pattern, it determines that it's not of type rollup
     if (indexPatternsUtils.isDefault(indexPattern)) {
-      searchSource.setField(
-        'filter',
-        data.query.timefilter.timefilter.createRelativeFilter(indexPattern)
-      );
+      searchSource.setField('filter', data.query.timefilter.timefilter.createFilter(indexPattern));
```

Comment:
I suggest adding this comment about this specific implementation detail:

```suggestion
      // Set the date range filter fields from timeFilter using the absolute format. Search sessions requires that it be converted from a relative range
      searchSource.setField('filter', data.query.timefilter.timefilter.createFilter(indexPattern));
```

Reply:
We can do the same thing when applying the relative time filter